nima/webserver/webserver/src/main/java/io/helidon/nima/webserver/http1/Http1ServerResponse.java

```diff
@@ -269,8 +274,11 @@ private void handleSinkData(Object data, MediaType mediaType) {
         }
     }
 
-    private static void writeHeaders(Headers headers, BufferData buffer) {
+    private static void writeHeaders(Headers headers, BufferData buffer, boolean validate) {
         for (HeaderValue header : headers) {
```

Comment:
I think this should be done as follows:
```
if (validate) {
   headres.forEach(HeaderValue::validate);
}

for (HeaderValue header : headers) {
  header.writeHttp1Header(buffer);
}
```

Reply:
Updated based on comment. After rebase, it will look like this:

```
         if (validate) {
            headers.forEach(Header::validate);
        }
        for (Header header : headers) {
            header.writeHttp1Header(buffer);
        }
```